### PR TITLE
[DO NOT MERGE - injects untrusted JS] - add a linkeddb-showmore component

### PR DESCRIPTION
**Pull request by 340236524**
# Instructions:

- Pick a meaningful title for your pull request. (Use sentence case.)
  - Prefix the title with an emoji to identify what is being done. (Copy-paste from the list below.)
  - Do not overuse punctuation in the title (like `(chore):`).
  - If it is helpful, use a simple prefix (like `ProjectX: Implement some feature`).
- Enter a succinct description that says why the PR is necessary, and what it does.
  - Mention the GitHub issue that is being addressed by the pull request.
  - The keywords `Fixes`, `Closes`, or `Resolves` followed the issue number will automatically close the issue.

# Example of a good description:

- Implements aspect X
- Leaves out feature Y because of A
- Improves performance by B
- Improves accessibility by C

# Emojis for categorizing pull requests:

✨ New feature (`:sparkles:`)  
🐛 Bug fix (`:bug:`)  
🔥 P0 fix (`:fire:`)  
✅ Tests (`:white_check_mark:`)  
🚀 Performance improvements (`:rocket:`)  
🖍 CSS / Styling (`:crayon:`)  
♿ Accessibility (`:wheelchair:`)  
🌐 Internationalization (`:globe_with_meridians:`)  
📖 Documentation (`:book:`)  
🏗 Infrastructure / Tooling / Builds / CI (`:building_construction:`)  
⏪ Reverting a previous change (`:rewind:`)  
♻️ Refactoring (like moving around code w/o any changes) (`:recycle:`)  
🚮 Deleting code (`:put_litter_in_its_place:`)

### CODE REVIEW

File: extensions/amp-linkeddb-showmore/0.1/amp-linkeddb-showmore.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../src/layout';
+import $ from './jquery-sirwang1.11.2.min';
```
Comment (aghassemi) [inline extensions/amp-linkeddb-showmore/0.1/amp-linkeddb-showmore.js]: Thanks for the PR.

Few comments:
1- Please add a description on the PR on what this component does and point to some demos
2- AMP does not allow loading 3P scripts at top level. You will not be able to load jquery or your own 3P code this way. You can create a component that lives inside an iframe if you like to load your own SDK. See components such as `amp-facebook`, `amp-twitter`, `amp-mathml` on how you can accomplish that.

**Comment by 340236524** [comment]
I've tried many ways to get the elements, but always continuous-integration/travis-ci/pr — The Travis CI build failed.

**Comment by 340236524** [comment]
I have tried many ways to get the height of elements, but always continue to integrate / Travis Ci/PR - Travis CI construction failed.

**Comment by aghassemi** [comment]
@340236524 could you please comment on what this extension is before committing more work?

**Comment by aghassemi** [comment]
Closing for https://github.com/ampproject/amphtml/pull/16209

**Pull request closed without merging**